contracts/Aavegotchi/facets/AavegotchiGameFacet.sol

```diff
@@ -328,4 +325,30 @@ contract AavegotchiGameFacet is Modifiers {
     function isAavegotchiLocked(uint256 _tokenId) external view returns (bool isLocked) {
         isLocked = s.aavegotchis[_tokenId].locked;
     }
+
+    ///@notice Allow the current owner/borrower of an NFT to reduce kinship while channelling alchemica
+    ///@dev will revert if the gotchi kinship is too low to channel or if the lending listing does not enable channeling
+    ///@param _gotchiId Id of the Gotchi used to channel
+    function reduceKinshipViaChanneling(uint32 _gotchiId) external {
```

Comment:
Not sure this should be bundled together as one single function. Seems like it would be cleaner and more modular to have kinshipBurning as a function separate from channeling.

Reply:
do you mean having `kinshipBurning` seperate from `interacting`?